chat-plugins/scavengers.js

```diff
@@ -797,8 +797,8 @@ let commands = {
 			str += Chat.html`<tr><td>Completed</td><td>${game.completed.length ? game.completed.map(pl => pl.name).join(", ") : 'None'}`;
 			this.sendReply('|raw|' + str + "</table></div>");
 		} else {
-			let elapsed = Date.now() - game.startTime;
-			this.sendReplyBox(`The current ${game.gameType ? `<em>${game.gameType}</em> ` : ""}scavenger hunt by <em>${Chat.escapeHTML(Chat.toListString(game.hosts.map(h => h.name)))}${(game.hosts.some(h => h.userid === game.staffHostId) ? '' : ` (started by - ${Chat.escapeHTML(game.staffHostName)})`)}</em> has been up for: ${Chat.toDurationString(elapsed, {hhmmss: true})}<br />Completed (${game.completed.length}): ${game.completed.map(u => Chat.escapeHTML(u.name)).join(', ')}<br />`);
+			const elapsed = Date.now() - game.startTime;
+			this.sendReplyBox(Chat.html`The current ${game.gameType ? `<em>${game.gameType}</em> ` : ""}scavenger hunt by <em>${Chat.toListString(game.hosts.map(h => h.name))}${(game.hosts.some(h => h.userid === game.staffHostId) ? '' : ` (started by - ${Chat.escapeHTML(game.staffHostName)})`)}</em> has been up for: ${Chat.toDurationString(elapsed, {hhmmss: true})}<br />Completed (${game.completed.length}): ${game.completed.map(u => Chat.escapeHTML(u.name)).join(', ')}<br />`);
```

Comment:
>Chat.escapeHTML(game.staffHostName)
>Chat.escapeHTML(u.name)

You don't need to do that for either of those because this template string starts with ``Chat.html``

Reply:
woops thought i removed them all

Reply:
You can't pass in `<em>${game.gameType}</em>` inside a template string...